src/Slices/Notification/UI/Center/Item.tsx

```diff
@@ -0,0 +1,100 @@
+import React from "react";
+import {
+  DataListAction,
+  DataListCell,
+  DataListItem,
+  DataListItemCells,
+  DataListItemRow,
+  variantIcons,
+} from "@patternfly/react-core";
+import styled from "styled-components";
+import { MomentDatePresenter } from "@/UI/Utils";
+import { Model, Severity } from "@S/Notification/Core/Model";
+import {
+  getColorForVisualSeverity,
+  getSeverityForNotification,
+  VisualSeverity,
+} from "@S/Notification/UI/Utils";
+import { ActionList } from "./ActionList";
+
+interface Props {
+  notification: Model;
+  onUpdate(): void;
+}
+
+export const Item: React.FC<Props> = ({ notification, onUpdate }) => {
+  const visualSeverity = getSeverityForNotification(notification.severity);
+  return (
+    <CustomItem
+      aria-label="NotificationItem"
+      $read={notification.read}
+      $severity={visualSeverity}
+    >
+      <DataListItemRow>
+        <DataListItemCells
+          dataListCells={[
+            <DataListCell key="main">
+              <div>
+                <Icon {...notification} />
+                <Title $read={notification.read}>{notification.title}</Title>
+              </div>
+              <p>{notification.message}</p>
+              <Timestamp>
+                {new MomentDatePresenter().get(notification.created).relative}
```

Comment:
I think it would be useful to show the full time in a tooltip as well, similarly to other pages

Reply:
I have added it to the center page. I have not added it to the drawer because it is not in our control there.

Reply:
Yes, it's fine to only have it on the center page :+1: